### ISSUE TITLE
Unknown escape sequence in git ignore pattern

### ISSUE THREAD

**Issue by dlukeomalley** [returntocorp/bento#187]
bento init fails when `.gitignore` contains the following pattern:

```shell
/\.project
```

```shell
$ bento init

...

Creating default configuration at .bento.yml
✘ There was an exception Unknown escape sequence '\.' in ignore pattern '/\.project'
```

This was expected to work because it's valid `.gitignore` syntax, even though it appears to be a NOOP.

**Comment by ulziibay** [comment]
I cannot repro this on my project. It seems resolved!

**Comment by dlukeomalley** [comment]
@ulziibay I just reproduced and added additional steps and context above. Apologies for lack of them before.

**Comment by DrewDennison** [comment]
@nbrahms fixed in returntocorp/bento-core/pull/408
